lib/src/http/common/request.dart

```diff
@@ -707,4 +704,51 @@ abstract class CommonRequest extends Object
     didSucceed = true;
     return response;
   }
+
+  Duration _createExponentialBackOff() {
+    int backOffInMs = autoRetry.backOff.interval.inMilliseconds *
+        pow(defaultExponentialMultiplier, autoRetry.numAttempts);
+    backOffInMs =
+        min(autoRetry.backOff.maxInterval.inMilliseconds, backOffInMs);
+
+    if (autoRetry.backOff.withJitter != null &&
+        autoRetry.backOff.withJitter == true) {
```

Comment:
(indeed)
I can't remember why I avoided that in Dart, but I tested it in Dart 1.12 and it works.